src/raster_types.jl

```diff
@@ -12,7 +13,24 @@ function Raster(dataset::GDALDatasetH)
                     _getrastercount(dataset)) # number of bands
     finalizer(raster, closeraster)
     raster
-end 
+end
+
+function Base.show(io::IO,raster::Raster)
+     if (raster.dataset == C_NULL)
+         println(io, raster.dataset)
+         return null;
+     end
+     # datatypes
+     if raster.nband==1
+         print("$(raster.height) x $(raster.width) (rows, cols) Single-Band Raster of type $(dtype(_getrasterband(raster.dataset, 1)))")
```

Comment:
these should be printing to `io` -- right now, they print to `STDOUT`, even when the function is passed something else via `io`.